modules/default/alert/styles/notificationFx.css

```diff
@@ -1,7 +1,9 @@
+/* stylelint-disable number-max-precision */
```

Comment:
can we get rid of that ignore?

Reply:
Gladly. I see two approaches:

1. Reduce the values by one digit in the animations. I don't think it has any visible impact.
2. Increase the value for number-max-precision by one in the .stylelintrc.json.

What do you think?

Reply:
I would go for 1) since 
- like you said it wouldnt probably impact the looks
- we would stay closer to the mainline stylelint rules

Reply:
I also think this is the better choice. I re-enabled the rule and performed the rounding